src/screens/TestCases/Modal/CustomFilterModal.jsx

```diff
@@ -20,7 +20,11 @@ const CustomFilterModal = ({
   type,
   handleApplyButton,
   filterData,
-  localDispatch
+  localDispatch,
+  handleClearAllButton,
+  handleSearchChange,
+  handleClearAllFilter,
```

Comment:
Not used than remove it.
Avoid rebundancy

Reply:
resolved